components/search/index.css

```diff
@@ -25,49 +35,44 @@
 	--spectrum-search-to-help-text: var(--spectrum-help-text-to-component);
 	--spectrum-search-top-to-text: var(--spectrum-component-top-to-text-100);
 	--spectrum-search-bottom-to-text: var(--spectrum-component-bottom-to-text-100);
-	--spectrum-search-border-radius: var(--spectrum-corner-radius-100);
-	--spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-100);
-
-	/* Focus Indicator */
-	--spectrum-search-focus-indicator-thickness: var(--spectrum-focus-indicator-thickness);
-	--spectrum-search-focus-indicator-gap: var(--spectrum-focus-indicator-gap);
-	--spectrum-search-focus-indicator-color: var(--spectrum-focus-indicator-color);
```

Comment:
Definitely open to keeping these focus indicator properties in. I figured since search field is basically extending textfield, maybe these weren't necessary though.